### DISCUSSION TITLE
How to make PR title for single update look like PR title for group?

### DISCUSSION THREAD

**Discussion by vitoyucepi** [renovatebot/renovate#9580] [Request Help]
## Summary
I want to create groups based on different paths.
So I created some `packageRules`
```json
{
  "extends": [
    "config:base",
  ],
  "pinDigests": true,
  "prCreation": "not-pending",
  "stabilityDays": 1,
  "packageRules": [
    {
      "matchPaths": ["service1/docker-compose{,.*}.{yaml,yml}"],
      "matchDatasources": ["docker"],
      "groupName": "Service1 containers",
    },
    {
      "matchPaths": ["service2/docker-compose{,.*}.{yaml,yml}"],
      "matchDatasources": ["docker"],
      "groupName": "Service2 containers",
    },
    {
      "matchPaths": ["service3/docker-compose{,.*}.{yaml,yml}"],
      "matchDatasources": ["docker"],
      "groupName": "Service3 containers",
    }
  ]
}
```
If there are more then one update in group then group PR is created with name `Update Service1 containers` and group suffix `(service1/container1 database/image2 support/image3)`.
But if there's only one update in group the PR will be look like `Update service2/image2 Docker tag to v1.2.3`.

## Question
How can I make single update PR look like group PR?
Or maybe is it possible to create a group PR even if a single package awaits update?
1. I want to see the group name
2. I want to see container image name that was updated

**Comment by rarkins** [comment]
I haven't tested this, but you could try adding `"commitMessageTopic": "{{{groupName}}}"` to the applicable package rules

**Reply by rarkins** [reply]
Default behavior is to use the group branch name always, but only use the group commit message and PR title if > 1 unique updates are in the branch

**Reply by vitoyucepi** [reply]
Not working like expected.
Now non-groups are `Update Docker tag to v1.2.3`

**Reply by rarkins** [reply]
Sorry, can't think of any way to do it reliably, but you could experiment with commitMessage fields. If you have one dependency in a group then the pr title will reflect the one dependency's name and version.

**Reply by vitoyucepi** [reply]
Tried
```json
    {
      "matchPaths": ["service2/docker-compose{,.*}.{yaml,yml}"],
      "matchDatasources": ["docker"],
      "groupName": "Service2 containers",
      "commitMessageTopic": "Service2 containers",
      "commitMessageExtra": "",
      "commitMessageSuffix": "(`{{{depName}}}`)",
    }
```
For non group PR title is ```Update Service2 containers (`service2/image2`)``` with no suffix like in groups. Backticks and braces are treated literaly.

**Reply by vitoyucepi** [reply]
Probably the best result I've get was
```json
    {
      "matchPaths": ["service2/docker-compose{,.*}.{yaml,yml}"],
      "matchDatasources": ["docker"],
      "groupName": "Service2 containers",
      "commitMessageTopic": "Service2 containers",
      "commitMessageExtra": "({{{depName}}})"
    }
```

**Reply by rarkins** [reply]
Adding the depName might not work well once you have >1 dependency in a branch

**Reply by vitoyucepi** [reply]
This works well for me because groups have no `commitMessageExtra` in template.

But it's not the solution. I want to add suffix like in groups.
